### PR TITLE
Add model to reactions

### PR DESCRIPTION
**Pull request by jesusMava**
### Description:
The user should be able to add one reaction (real or fake) by curiosity

### Solution:

Add model to be able to add reactions. 
with this model, we'll be able to manage the reactions of each user over each curiosity, we don't need to save a history only that each record doesn't save twice 
image

### CODE REVIEW

File: db/migrate/20220127002337_create_reactions.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreateReactions < ActiveRecord::Migration[7.0]
+  def change
+    create_table :reactions do |t|
+      t.boolean :statement
```
Comment (itsmeurbi) [inline db/migrate/20220127002337_create_reactions.rb:4]: Is it posible for a reaction record not o have a `statement`?
If not, then probably there should be a valuation at db level as well

File: test/models/reaction_test.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  def user
+    @user ||= create(:user)
+  end
+
+  setup do
+    @reaction = create(:reaction, user: user, curiosity_card: curiosity)
+  end
+
+  test 'should exit only one register by action' do
+    reaction_again = build_stubbed(:reaction, user: user, curiosity_card: curiosity)
```
Comment (itsmeurbi) [inline test/models/reaction_test.rb:19]: Remember the empty spaces between each 'block'

**Review by itsmeurbi — CHANGES_REQUESTED** [review]
Rather than my comments LGTM!

File: db/migrate/20220127002337_create_reactions.rb (hunk 1 above)
Comment (jesusMava) [reply db/migrate/20220127002337_create_reactions.rb:4]: yes, it's possible for instance, I can have this 

| id_user | curiosity_id | statement |
| ----------- | ----------- | ----------- |
| 1 | 2 | niil |
| 1 | 3 | true |

Comment (itsmeurbi) [reply db/migrate/20220127002337_create_reactions.rb:4]: Having `statement` as nil...what does the record mean?

Comment (jesusMava) [reply db/migrate/20220127002337_create_reactions.rb:4]: Well if a record is nil means that the user delete the true or false actions and now the record doesnt have either or is it a better idea deleted it?  🤔

**Review by itsmeurbi — APPROVED** [review]
LGTM!

**Pull request merged**